components/common/src/ome/util/prefs.java

```diff
@@ -5,6 +5,8 @@
  *   Use is subject to license terms supplied in LICENSE.txt
  */
 
+package ome.util;
```

Comment:
This move makes this class unusable in its previous way:

```
$ bin/omero config -h
...
    old               Delegate to the old configuration system using Java preferences
```
You might as well delete it if you are moving it.